### DISCUSSION TITLE
How to remove `friendly-snippets` ?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [NvChad/NvChad#893] [Q&A] [answered]
I have my own set of snippets and I don't need `friendly-snippets`. But apparently, it's a dependency of `nvim-cmp` and since I removed it I get this error after running `:PackerCompile`:
```
async.lua:20: Error in coroutine: ......e.lua:592: Dependency friendly-snippets for { "nvim-cmp" } not found
```
How could I remove this plugin without changing files outside the `custom` folder?

**Comment by mattrebor** [comment]
@amirhhashemi Did you figure out a way? 

I've been looking but I'm guessing there's no way to prevent the friendly-snippets from loading without modifying `init.lua` (the main one and not the one in  `custom` folder)

**Reply by siduck** [reply, marked as answer]
@mattrebor @amirhhashemi 
```lua
{
    "rafamadriz/friendly-snippets", enabled = false
  },
  ```
  
  @amirhhashemi sorry i got no notification of this discussion at all!! so couldnt help with it, If you get no replies for 1-2 days then just ping my username directly
